File: RecommendationApp.API/Models/Team.cs
namespace RecommendationApp.API.Models
{
    public class Team
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Country { get; set; }
        public string About { get; set; }
        public double Rating { get; set; }

        public Team(int id, string name, string country, string about, double rating)
        {
            Id = id;
            Name = name;
            Country = country;
            About = about;
            Rating = rating;
        }
    }
}

File: RecommendationApp-Client/src/app/_models/team.ts
import { Player } from '../player';
import { BestMap } from './bestMap';

export interface Team{
    profileId: number;
    name:string;
    country:string;
    about:string;
    rating:number;
    //createdAt:?
    players:Player[],
    bestMaps:BestMap[]
}